reconcile/utils/aws_api.py

```diff
@@ -715,11 +715,12 @@ def _get_assume_role_session(self, account_name: str, assume_role: str,
 
     # pylint: disable=method-hidden
     def _get_assumed_role_client(self, account_name: str, assume_role: str,
-                                 assume_region: str) -> EC2Client:
+                                 assume_region: str,
+                                 client_type: str = 'ec2') -> EC2Client:
         assumed_session = self._get_assume_role_session(account_name,
                                                         assume_role,
                                                         assume_region)
-        return assumed_session.client('ec2')
+        return assumed_session.client(client_type)
```

Comment:
```
reconcile/utils/aws_api.py:723: error: No overload variant of "client" of "Session" matches argument type "str"
```

Reply:
removed the type hint to get rid of this (https://coreos.slack.com/archives/GGC2A0MS8/p1637668588271300)